src/include/cluster/dialogs.rb

```diff
@@ -1242,8 +1241,13 @@ def csync2_status
         return 2
       end
       #check the firewall whether csync2 port was blocked.
-      tcp_ports = []
-      tcp_ports = SuSEFirewallServices.GetNeededTCPPorts("service:cluster")
+      begin
+        firewalld_cluster = firewalld.find_service("cluster")
+        tcp_ports = firewalld_cluster.tcp_ports
+      rescue Y2Firewall::Firewalld::ServiceNotFound
+        tcp_ports = []
+      end
+
```

Comment:
Should be "rescue Y2Firewall::Firewalld::Service::NotFound" instead of "rescue Y2Firewall::Firewalld::ServiceNotFound"

Reply:
yep, it should